webrtc.html

```diff
@@ -4043,6 +4043,19 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <p>If the sender is not already active and the PeerConnection is in the stable state,
             this will set the <code>negotiationneeded</code> flag.
           </dd>
+          
+          <dt>void activateReceiver()</dt>
+
+          <dd>
+            <p>The <dfn id="dom-rtptransceiver-activatereceiver"><code>activateReceiver</code></dfn>
+            method activates the <code>RTCRtpTransceiver.receiver</code> such that future calls
+            to <code>createOffer</code> and <code>createAnswer</code> mark the corresponding
+            media description as <code>sendrecv</code> or <code>recvonly</code>, as defined in
+            [[!JSEP]].
+
+            <p>If the receiver is not already active and the PeerConnection is in the stable state,
+            this will set the <code>negotiationneeded</code> flag.
+          </dd>
```

Comment:
I was hoping we would need only one "set horrible" (activateSender()) and not two (activateReceiver()).  

Does this accomplish anything other than a way to tell the sending side "please stop sending me this track" without rejecting the whole m-line?